transport-native-epoll/src/main/c/io_netty_channel_epoll_Native.c

```diff
@@ -115,17 +119,33 @@ jint JNI_OnLoad(JavaVM* vm, void* reserved) {
     if ((*vm)->GetEnv(vm, (void**) &env, JNI_VERSION_1_6) != JNI_OK) {
         return JNI_ERR;
     } else {
-        if (netty_unix_errors_JNI_OnLoad(env) == JNI_ERR) {
+        // Load the prefix to use when looking for Netty classes
+        jclass systemCls = (*env)->FindClass(env, "java/lang/System");
+        if (systemCls != NULL) {
```

Comment:
It is not essential to function, but I can throw an error so we don't silently swallow it.